test/integ-new/main_new_test.go

```diff
@@ -297,3 +370,108 @@ func testSendHTTPReq(ctx context.Context, tb testing.TB, uri, token string) *htt
 
 	return resp
 }
+
+func testCallKeyRotationEndpoint(ctx context.Context, tb testing.TB) {
```

Comment:
Maybe we can combine testSendHTTPReq, we don't need two similar helper func.

Reply:
Combined.